### ISSUE TITLE
sense-key

### ISSUE THREAD

**Issue by arademaker** [own-pt/own-en-legacy#19]
bug in the sense key. The sense-key cannot be a word concatenated with a number, see the case of http://wnpt.brlcloud.com/wn/search?term=1000000000000. We should use a explicit separator like `#` instead.

**Comment by fcbr** [comment]
We could follow PWN's lexicographer files convention:

```
fcbr@FCBR-TP:~/work/wn/dist/WNgrind-3.0/dict/dbfiles$ grep 1000000000000 noun.*
noun.quantity:{ [ billion1, adj.all:cardinal^billion1,+ ] one_million_million, 1000000000000", large_integer,@ noun.location:Britain,;r (the number that is represented as a one followed by 12 zeros; in the United Kingdom the usage followed in the United States is frequently seen) }
noun.quantity:{ trillion, one_million_million1, 1000000000000"1, large_integer,@ noun.location:US,;r noun.location:France,;r (the number that is represented as a one followed by 12 zeros; "in England they call a trillion a billion") }
```

**Comment by arademaker** [comment]
Other cases in the original PWN files:

```
$ egrep "[0-9]+\"[0-9]+" *
noun.quantity:{ trillion, one_million_million1, 1000000000000"1, large_integer,@ noun.location:US,;r noun.location:France,;r (the number that is represented as a one followed by 12 zeros; "in England they call a trillion a billion") }
noun.time:{ Saint_Agnes's_Eve, January_20"1, Christian_holy_day,@ January,#p (a Christian holy day) }
noun.time:{ Solemnity_of_Mary, January_1"1, holy_day_of_obligation,@ January,#p noun.group:Roman_Catholic,;c ((Roman Catholic Church) a holy day of obligation) }
noun.time:{ Circumcision, Feast_of_the_Circumcision, January_1"2, feast_day,@ noun.group:Roman_Catholic,;c noun.group:Anglican_Church,;c ((Roman Catholic Church and Anglican Church) feast day celebrating the circumcision of Jesus; celebrated on January 1st) }
```

**Comment by arademaker** [comment]
The documentation of http://wordnet.princeton.edu/wordnet/man/wninput.5WN.html says

> word may be entered in any combination of upper and lower case unless it is in an adjective cluster. A collocation is entered by joining the individual words with an underscore character (_ ). Numbers (integer or real) **may** be entered, **either by themselves** or **as part of a word string**, by following the number with a double quote (" ).

**Comment by arademaker** [comment]
We have also the cases when the word ends with double quote:

```
$ egrep "January_[0-9]+" *
noun.time:{ Inauguration_Day, January_20", day1,@ January,#p (the day designated for inauguration of the United States President) }
noun.time:{ New_Year's_Day, New_Year's, January_1", legal_holiday,@ January,#p (the first day of the year) }
noun.time:{ Robert_E_Lee's_Birthday, Robert_E_Lee_Day, Lee's_Birthday, January_19", day1,@ (celebrated in southern United States) }
noun.time:{ Saint_Agnes's_Eve, January_20"1, Christian_holy_day,@ January,#p (a Christian holy day) }
noun.time:{ Solemnity_of_Mary, January_1"1, holy_day_of_obligation,@ January,#p noun.group:Roman_Catholic,;c ((Roman Catholic Church) a holy day of obligation) }
noun.time:{ Circumcision, Feast_of_the_Circumcision, January_1"2, feast_day,@ noun.group:Roman_Catholic,;c noun.group:Anglican_Church,;c ((Roman Catholic Church and Anglican Church) feast day celebrating the circumcision of Jesus; celebrated on January 1st) }
noun.time:{ Epiphany, Epiphany_of_Our_Lord, Twelfth_day, Three_Kings'_Day, January_6", Christian_holy_day,@ January,#p (twelve days after Christmas; celebrates the visit of the three wise men to the infant Jesus) }
```

**Comment by arademaker** [comment]
In the original lex files from PWN we also have 

```
adj.all:{ Y2"K_compliant, (prepared to accurately process date and time data between and into the 20th and 21st centuries) }
```

**Comment by arademaker** [comment]
I followed the idea of PWN but I tend to prefer the use of explicit separator.

**Comment by fcbr** [comment]
we still have certain words ending with digits without the proper escape.

```
w: Cambrian_Stage_4
```

**Comment by arademaker** [comment]
one more case fixed in 77eb6ddf4abe8f3e5e918ddabecea1bcb8d5730d
